src/base/system.c

```diff
@@ -1647,6 +1702,15 @@ int fs_is_dir(const char *path)
 #endif
 }
 
+time_t fs_getmtime(const char *path)
+{
+	struct stat sb;
+	if(stat(path, &sb) == -1)
```

Comment:
This probably doesn't work for older vs versions.

Reply:
Should we support older vs versions? Its in ddnet code for a while now and I haven't noticed any compile complaints about this.